partner_statement/report/activity_statement.py

```diff
@@ -48,7 +48,7 @@ def _initial_balance_sql_q1(self, partners, date_start, account_type):
                 WHERE l2.date < %(date_start)s
             ) as pc ON pc.credit_move_id = l.id
             WHERE l.partner_id IN %(partners)s
-                AND l.date < %(date_start)s AND not l.blocked
```

Comment:
why blocked is removed?

Reply:
@AurelijaNorvaisaite block [was removed](https://github.com/odoo/odoo/commit/67dc71588751c9f115e09959d774686c2207a1f9) on Odoo 18.0.